### PR TITLE
separate kernel installation and download

### PR DESCRIPTION
**Pull request by wizardofoz123**
This continues work on FASTMOVING-591.

This patch does following:

* If a kernel download fails, it's likely an infrastructure issue, so we
want to report warn/aborted.
* If a kernel package fails to install after being downloaded, it's
  likely a distro issue, so we abort the recipe as well, but report fail
  instead.

Signed-off-by: Jakub Racek <jracek@redhat.com>

### CODE REVIEW

**Comment by wizardofoz123** [comment]
thou shall test this, @cki-bot !

File: distribution/kpkginstall/runtest.sh [hunk 1]
```diff
@@ -197,6 +197,29 @@ function copr_prepare()
   return 0
 }
 
+function download_install_package()
+{
+  $YUM install --downloadonly -y $1 | tee -a ${OUTPUTFILE}
```
Comment (veruu) [inline distribution/kpkginstall/runtest.sh]: Can we have the stderr redirected here too, or does it cause trouble? It is useful for debugging to have the message logged directly

Comment (wizardofoz123) [reply distribution/kpkginstall/runtest.sh]: np, added for both download & install

**Comment by wizardofoz123** [comment]
Thanks for review. Merging.

**Pull request merged**